### PR TITLE
add JpegQuality parameter

### PR DESCRIPTION
**Pull request by alaeri**
I added a jpegQuality parameter. 

I copied what was done with sensorSensitivity (except the selector, not needed here as jpegQuality is an int ranging from 0 to 100 for camera Api). I think I've set the default compression value to 95, I'm not sure I have understood everything about what I did as I was in a hurry. I tried to follow existing format conventions as I was doing it.

I integrated it to the sample and I have tested it with Camera1 (not tested with Camera2 yet)

### CODE REVIEW

**Comment by Diolor** [comment]
Hey @alaeri totally awesome you made this PR! I added few minor notes. Would be nice to have those fixed. I would also be nice to have a QA on this before merging but I don't have time rn to do

**Comment by alaeri** [comment]
Hey @Diolor  
Very glad you like the PR 👍 

I can't see the notes you added. What kind of QA can I do to help?

File: fotoapparat/src/main/java/io/fotoapparat/FotoapparatBuilder.java [hunk 1]
```diff
@@ -144,6 +145,14 @@ public FotoapparatBuilder sensorSensitivity(@NonNull SelectorFunction<Range<Inte
     }
 
     /**
+     * @param jpegQuality of the picture (1-100)
+     */
+    public FotoapparatBuilder jpegQuality(@NonNull Integer jpegQuality) {
```
Comment (dmitry-zaitsev) [inline fotoapparat/src/main/java/io/fotoapparat/FotoapparatBuilder.java]: Let's add `@IntRange` annotation for more robustness

File: sample/src/main/java/io/fotoapparat/sample/MainActivity.java [hunk 2]
```diff
@@ -207,14 +210,22 @@ public void onError(CameraException e) {
...
-        PhotoResult photoResult = fotoapparatSwitcher.getCurrentFotoapparat().takePicture();
+        final PhotoResult photoResult = fotoapparatSwitcher.getCurrentFotoapparat().takePicture();
 
         photoResult.saveToFile(new File(
                 getExternalFilesDir("photos"),
                 "photo.jpg"
         ));
 
         photoResult
+                .toPendingResult().whenAvailable(new PendingResult.Callback<Photo>() {
```
Comment (dmitry-zaitsev) [inline sample/src/main/java/io/fotoapparat/sample/MainActivity.java]: I guess you added this to test the feature, but I would remove it from the example to keep it simple. Most of the users probably don't care how big picture is :)

File: fotoapparat/src/main/java/io/fotoapparat/FotoapparatBuilder.java (hunk 1 above)
Comment (alaeri) [reply fotoapparat/src/main/java/io/fotoapparat/FotoapparatBuilder.java]: That's a great idea! 
done in https://github.com/Fotoapparat/Fotoapparat/pull/125/commits/41bec5dc0ac4168d16e7e0525d93354ebef223a9

File: sample/src/main/java/io/fotoapparat/sample/MainActivity.java (hunk 2 above)
Comment (alaeri) [reply sample/src/main/java/io/fotoapparat/sample/MainActivity.java]: That's why I had added it. Removed from the sample now: 
https://github.com/Fotoapparat/Fotoapparat/pull/125/commits/143a3ad47cf985c30987dd71955e6edb677b0830

**Comment by dmitry-zaitsev** [comment]
Thanks for your effort! This will go into the next release.

**Comment by alaeri** [comment]
Thank you. That's great news!

**Pull request merged**